src/index.ts

```diff
@@ -41,7 +41,7 @@ const plugin: JupyterFrontEndPlugin<void> = {
       label: 'Fileglancer',
       icon: FileglancerIcon,
       execute: () => {
-        console.log('Create React widget command executed');
+        console.log('Open fileglancer command');
```

Comment:
I changed this only for testing but I can revert it if any of you thinks I should